client/src/Auth/Auth.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Login from './components/Login';
+import SignUp from './components/SignUp';
+
+const Auth = () => {
+  const [loggedIn, setLoggedIn] = useState(false);
```

Comment:
@cgf5033 assuming "const [loggedIn, setLoggedIn] = useState(false);" is just here to be used later when auth api is set up, just want to confirm.